exporter/zipkin/opentelemetry-exporter-zipkin.gemspec

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/exporter/zipkin/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-exporter-zipkin'
+  spec.version     = OpenTelemetry::Exporter::Zipkin::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'Zipkin trace exporter for the OpenTelemetry framework'
+  spec.description = 'Zipkin trace exporter for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = ::Dir.glob('lib/**/*.rb') +
+               ::Dir.glob('thrift/gen-rb/**/*.rb') +
+               ::Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 2.5.0'
+
+  spec.add_dependency 'opentelemetry-api', '~> 0.15.0'
```

Comment:
I just realized both this and the other exporters depend on the SDK, not (just) the API.

Reply:
updated the other exporters as well